signin.js

```diff
@@ -0,0 +1,84 @@
+const emailInput= document.querySelector('.email-input');
+const pwdInput=document.querySelector('.pwd-input');
+const loginButton=document.querySelector('.button-sign');
+const checkEmail= document.querySelector('.check-email');
+const checkPwd= document.querySelector('.check-pwd');
+const eyeIcon=document.querySelector('.pwd-eye-off');
+
+function emailCheck(email_address) {		
+    const email_regex = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}$/i;
+    if (!email_regex.test(email_address)) return false; 
+    else return true;
```

Comment:
`RegExp.test()` 의 반환값은 boolean 타입이므로 아래와 같이 작성할 수도 있습니다

```suggestion
    return email_regex.test(email_address); 
```

참고: https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test#%EB%B0%98%ED%99%98_%EA%B0%92